src/program/lwaftr/run/run.lua

```diff
@@ -158,6 +158,7 @@ function run(args)
    end
 
    if opts.reconfigurable then
+      conf.alarm_notification = true
```

Comment:
Because the conf is a product of a yang schema I hesitate to do this.  Instead I would have setup.lua arrange to ensure that ARP is created with alarm notification enabled.